CharacterCreation/01_04_Species_Opaleites.txt

```diff
@@ -2,7 +2,18 @@
 Opaleites
 ==============================
 
-  Opaleites are an organic amphibious species that 
+    Opaleites are an organic amphibious species that evolved on a planet that 
+was mostly covered by marshes, seas and oceans. Their unique resperatory system
+lets them breathe under water as well as on land. An Opaleite's eyes are a 
+little better at seeing underwater then they are out of it.
+
+    Play as an Oplaite if you want to stick close to your team as if they were 
+your schoolmates. If your clan and where you came from is very important to you. 
+If you want to go anywhere in land, sea or air and see what there is to see in 
+this beautiful and diverse universe. Play as an Opalite if what you eat is very
+important to you. Play as a sub-species of Opalite if you want to dazzle the 
+world with your beautiful scales or be the toughest fish in the bay with your 
+bullet-resistant scales. 
 
 ===== Appearance/Anatomy =====
```

Comment:
Can we make the title for this section of Species files read "Appearance and Anatomy"? The "/" could imply that a) the section is about only one or the other, or b) we treat appearance and anatomy as interchangeable terms (which we shouldn't: anatomy is a scientific term focused on function, appearance is an aesthetic term focused on perception and social interaction).

Reply:
If we change that here, we need to change that in the rest of them in the Races-Push branch as well. I'd rather ship this as-is and then systematically update the rest in a subsequent issue so we can side-by-side each of the species.